### PR TITLE
Feature/sidebar improvements

### PR DESCRIPTION
**Pull request by weberjavi**
This Pr updates Sidebar content to match new designs
Designs: https://projects.invisionapp.com/d/main/default/#/console/15271576/318740046/preview

### CODE REVIEW

File: src/app/pages/root/group-card-list/group-card-list.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { mapStateToProps } from './group-card-list-selectors';
+import GroupCardComponent from './group-card-list-component';
+
+class GroupCardContainer extends Component {
+  render() {
+    return <GroupCardComponent {...this.props} />;
```
Comment (j8seangel) [inline src/app/pages/root/group-card-list/group-card-list.js]: We could remove the container logic if we are not using it.

File: src/app/pages/root/group-card-list/group-card-list-component.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                  onClick={() => this.toogleGroupActive(group[0].groupSlug)}
+                  theme={{ button: styles.toggleButton }}
+                >
+                  <Icon
+                    icon={chevronIcon}
+                    theme={{ icon: this.isGroupOpen(group[0].groupSlug) ? styles.icon : styles.iconCollapse }}
+                  />
+                </Button>
+              </div>
+              <CategoriesList categories={group} />
```
Comment (j8seangel) [inline src/app/pages/root/group-card-list/group-card-list-component.jsx]: I like more the approach of having a component which wraps the logic on the "accordion" and then we can reuse it by passing the children component. what do you think?

File: src/app/components/v2/dataset-combo/dataset-combo-component.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+              )}
+        </div>
+          )
+      }
+    </div>
+  );
+};
+
+DatasetComboComponent.propTypes = {
+  className: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply src/app/components/v2/dataset-combo/dataset-combo-component.jsx]: ⚡️ done

File: src/app/pages/root/categories-list/categories-list-component.jsx [hunk 4]
```diff
@@ -1,115 +1,94 @@
...
+      <div className={styles.featuredCategory} key={category.slug}>
+        <div style={{ backgroundImage: `url(${category.imageUrl})` }} className={styles.featuredImage} />
+        <div className={styles.featuredCategoryContentContainer}>
+          <div className={styles.categorySection}>
+            <h2 className={styles.categoryTitle}>{category.name}</h2>
+            {category.metadata && this.renderInfoButton(category)}
+          </div>
+          {category.datasets.map(dataset => (
+            <DatasetCombo
+              {...this.props}
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply src/app/pages/root/categories-list/categories-list-component.jsx]: ⚡️ done

File: src/app/components/v2/checkbox/checkbox.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import CheckboxComponent from './checkbox-component';
```
[the remark this answers is not in the document]
Comment (weberjavi) [reply src/app/components/v2/checkbox/checkbox.js]: ⚡️ done

**Comment by j8seangel** [comment]
Looks awesome! I've simplified the categories grouping selector take a look on the latest commit, please.

**Pull request merged**